### PR TITLE
[Map] Sync package with standartized country list

### PR DESCRIPTION
**Pull request by Rolikasi**
This PR was created alongside with [this one](https://github.com/unicef/publicgoods-candidates/pull/737) 
**Work done**
- Standardized existing country list
- removed country cleaner script
- fixed bug with build output. It turned out that after nextjs update one more error appeared on build.

Related to issue in [publicgoods-candidates](https://github.com/unicef/publicgoods-candidates/issues/726)

I tested changes on my machine. It looks fine. Kindly confirm if it runs correctly @lacabra 
cc: @nathanfletcher

### CODE REVIEW

File: packages/map/package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
   },
   "devDependencies": {
     "@next/eslint-plugin-next": "^11.1.2",
+    "chokidar": "^2.1.8",
```
Comment (lacabra) [inline packages/map/package.json:30]: You added this new dependency but it is not used anywhere that I can see. 
Is it really needed, or can it be removed?

**Review by lacabra — CHANGES_REQUESTED** [review]
Everything looks good other than that unused dependency. Can you take a look @Rolikasi? Then we can merge

File: packages/map/package.json (hunk 1 above)
Comment (Rolikasi) [reply packages/map/package.json]: This package was added by mistake. Thank you

**Review by lacabra — APPROVED** [review]
This looks good to me, thank you!

**Pull request merged**